root/edit/notes.tt

```diff
@@ -23,6 +23,11 @@
                   <textarea class="edit-note" rows="5" placeholder="[% l('Add an edit note') %]"
                       name="enter-vote.vote.[% index %].edit_note"></textarea>
               </div>
+            <p>
+                  [% l('HTML is not supported, but {url|some basic syntax} is available. Please do always keep the {coc|Code of Conduct} in mind when writing edit notes!',
```

Comment:
It would be nice to use the same wording as for [annotation formatting](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-08-22/root/annotation/edit.tt#L38) for the first sentence, e.g. “Edit notes support a limited set of wiki formatting options”.

Reply:
Oh, that does sound good. I struggled to find some decent wording here so I had gone with what the wiki had.